changelog/fragments/upgrades.yaml

```
@@ -0,0 +1,15 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Add requirement cloud.common
```

```suggestion
      Add requirement cloud.common for Ansible-based operators
```
Because the changelog is for all operator types

---

updated